app/templates/gulpfile.js

```diff
@@ -254,9 +255,9 @@ gulp.task('optimize', ['inject', 'test'], function() {
 
   var assets = $.useref.assets({searchPath: './'});
```

Comment:
not sure if this line should be similarly changed as the filters that follow as restore() is called later on this